core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.ts

```diff
@@ -312,7 +337,25 @@ angular.module('oppia').component('stateTranslation', {
         var activeContentId = (
           $scope.stateHints[newIndex].hintContent.contentId);
         TranslationTabActiveContentIdService.setActiveContent(
-          activeContentId, WRITTEN_TRANSLATION_TYPE_HTML);
+          activeContentId, TRANSLATION_DATA_FORMAT_HTML);
+      };
+
+      $scope.changeActiveRuleContentIndex = function(newIndex) {
+        if (ctrl.isTranslationTabBusy) {
+          StateEditorService.onShowTranslationTabBusyModal.emit();
+          return;
+        }
+        if ($scope.activeRuleContentIndex === newIndex) {
+          return;
+        }
+        const {
+          rule, inputName, contentId
+        } = $scope.interactionRuleTranslatableContents[newIndex];
+        const activeContentId = contentId;
+        const activeDataFormat = rule.inputTypes[inputName];
```

Comment:
Oh, why is this done differently to the same occurrence in the `onTabClick` function? Isn't this inputType?

Reply:
Whoops, this is a bug. Thanks for catching it! I'll fix it.